pallets/offchain-worker/src/lib.rs

```diff
@@ -127,6 +127,11 @@ pub mod pallet {
 
 			let index_in_session = TryInto::<usize>::try_into(block_number).map_or(query_session_length, |bn| bn % query_session_length);
 
+			// SBP M1 review: the offchain_worker function will be called with the best block,
+			// but it's possible that it can "skip" blocks if the node has been down some time.
+			// This implies that some claims might be not be fulfilled because of the call
+			// to clear_claim in on_finalize which is called reliably at every block.
+
```

Comment:
All nodes enabling the offchain-worker in Litentry network cooperate to get the result according to the claims.  There are multiple nodes for each claim according to the load balance algorithm. So we have other online nodes to deal with the claim if one node was down.

Reply:
Fair point. So it boils down to the load balancing algo to ensure that more than every claim is most of the time processed more than once.

Reply:
Yes, it guarantee at lease one ocw worker take each task.